ixofoundation/ixo-webclient
# feat: add sector filter

## Scope
https://nonaredteam.atlassian.net/browse/IXO-727

## Work Done
Added the ability to filter entities by sector
Added the ability to set the visibility of the filter
Update the header override according to the sector selected

## Steps to Test
Run the app, go to `https://app_uat.ixo.world/entities/select?type=Cell&categories=[{%22name%22:%20%22Cell%20Type%22,%20%22tags%22:%20[%22Index%22,%20%22Relayer%22,%20%22Portal%22]},%20{%22name%22:%20%22Stage%22,%20%22tags%22:%20[%22Forming%22]}]&sector=Oracle%20Launchpad` and make sure the sector filter is set by checking the redux state. Make sure that resetting the filters does not change this value
